pkg/controllers/gateway/params_test.go

```diff
@@ -37,14 +35,14 @@ func TestGetParams(t *testing.T) {
 						Group:     "",
 						Kind:      "ConfigMap",
 						Name:      "test-params",
-						Namespace: testutil.Pointer(gatewayv1beta1.Namespace("test-ns")),
+						Namespace: testutil.Pointer(gatewayv1beta1.Namespace(testutil.Namespace)),
 					},
 				},
 			},
 			paramsObj: &corev1.ConfigMap{
 				ObjectMeta: metav1.ObjectMeta{
 					Name:      "test-params",
```

Comment:
As long as the name is the same it should be fine, but again - could it be defined somewhere once and then the variable reused?